src/js/common/global.js

```diff
@@ -1 +1,2 @@
 export const WINDOW_WIDTH_MD = 768
+export const targetElement = document.querySelector( '#bodyLock' )
```

Comment:
No, here now you are exporting only 1 hard-coded value of targetElement. The main point here is to have an ability to set and get any element as targetElement. Not only this one. You need to add getter and setter functions here.